INIT.md

```diff
@@ -0,0 +1,11 @@
+## Plan
+1. Add option `current_environment` to __Anyway::Settings__ class. Provide default value for this, maybe `ENV["RAILS_ENV"]`?
+2. Change method __Anyway::Loaders::YAML.call__. Add method `environmental?` and fetch all values from __Anyway::Settings.current_environment__ config file's section. Merge default values into resulting config keys.
+3. Delete files __Rails::Loaders::YAML__ and its tests.
+4. Add specs for environments in *spec/loaders/yaml_spec.rb*. Also add testcases for rails.
```

Comment:
I think, at this point, it would make sense to re-use the existing specs—that would guarantee backward compatibility (again).
So, just replacing `Anyway::Rails::Loaders::YAML.call` with `Anyway::Loaders::YAML.call` in the spec shouldn't break them.